wgsl/index.bs

```diff
@@ -4494,8 +4513,7 @@ the test name.
           module.
 * v-0021: Cannot re-assign a constant.
 * v-0022: Global variables must have a storage class.
-* v-0023: Entry point functions accept no parameters.
-* v-0024: Entry point functions return void.
+* v-0023: Entry point IO types must only be used on entry points.
```

Comment:
I think we're trying to phase out this section.   It's a bunch of second entries for rules elsewhere in the spec, and this will get out of date.

Please add a TODO: in the "Function declaration" section to say that 
formal arguments to entry points must be IO-shareable.
And that the return type of an entry point must be void or IO-shareable.

Reply:
The "Entry point declaration" section already has this text, which I think covers it?
> The parameters of an entry point have to be within Entry point IO types. They have in storage class. The return type of an entry point has to be of an Entry point IO type, or void.

I've removed the v-0023 error that I previously added.

Reply:
You're right. That covers it.
Thanks for removing the v-0023